### DISCUSSION TITLE
Docker-Compose File Used for Example Reverse Proxy

### DISCUSSION THREAD

**Discussion by sniperwhg** [Hubro/clipface#68] [Q&A] [answered]
I was wondering if the .yaml file could be provided for the example NGINX reverse proxy?

**Comment by Hubro** [comment, marked as answer]
What YAML file do you mean? :thinking: Are you talking about docker-compose.yaml? I don't use docker compose so I don't have an example of that.

**Reply by sniperwhg** [reply]
Got it, I was under the impression that the NGINX Docker Container in the example was launched via Docker-Compose instead of regular docker.
